test/testrunner.py

```diff
@@ -374,7 +381,7 @@ def main():
         os.chdir(os.path.join("test", "dlc_tests"))
 
         # Pull images for necessary tests
-        if specific_test_type == "sanity":
+        if "sanity" in specific_test_type or specific_test_type == "telemetry":
```

Comment:
why do you need to do this for the telemetry ec2 tests? iirc this pulls images to host, and the telemetry tests should be running on ec2

Reply:
Ah, I wasn't sure if tests running on ec2 needed that so I put that in to maintain previous behaviors. I'll remove it.